### PR TITLE
output csvs from growth rate not folders

### PR DESCRIPTION
**Pull request by colindaven**

### CODE REVIEW

File: tutorial/tutorial.txt [hunk 1]
```diff
@@ -1,133 +0,0 @@
...
-
-#1 edit the location "path_we" in the following script, then run it to collect the Wochenende files and directories needed.
-nano get_wochenende.sh 
-# Now run the script to set the needed scripts and directories for the pipeline
-bash get_wochenende.sh
-
-
-#2. Wochenende needs sample names like sputum_R1.fastq and sputum_R2.fastq to run correctly. So rename your files.
-srun gunzip *.fastq.gz
-rename 's/_001//g' *.fastq
```
Comment (LisaHollstein) [inline tutorial/tutorial.txt:29]: should we add this information in the Wiki?

File: nf_wochenende.nf [hunk 2]
```diff
@@ -224,14 +224,6 @@ workflow {
     //haybaler(reporting.out.us_csvs.collect().flatten())
```
Comment (LisaHollstein) [inline nf_wochenende.nf:224]: I always forget to remove this unnessecary line, could you do that?

File: tutorial/tutorial.txt (hunk 1 above)
Comment (colindaven) [reply tutorial/tutorial.txt:29]: yep, done, good point

File: nf_wochenende.nf (hunk 2 above)
Comment (colindaven) [reply nf_wochenende.nf]: done

Comment (LisaHollstein) [reply nf_wochenende.nf]: Thanks!

**Review by LisaHollstein — APPROVED** [review]

**Pull request merged**